doc/source/whatsnew/v1.3.0.rst

```diff
@@ -482,6 +482,7 @@ Other
 - Bug in :class:`Styler` where rendered HTML was missing a column class identifier for certain header cells (:issue:`39716`)
 - Bug in :meth:`Styler.background_gradient` where text-color was not determined correctly (:issue:`39888`)
 - Bug in :meth:`DataFrame.equals`, :meth:`Series.equals`, :meth:`Index.equals` with object-dtype containing ``np.datetime64("NaT")`` or ``np.timedelta64("NaT")`` (:issue:`39650`)
+- Bug in :class:`Styler` copy methods which were not updated for recently introduced attributes (:issue:`39708`)
```

Comment:
can move this up to where other `Styler` Items are listed

Reply:
this may not be necessary if this changes were in 1.3 (which they might be); you can add this issue number with another change for Styler (e.g. where the attributes were changed)

Reply:
this has to be 1.3, since tooltips is the most recent and thats in 1.3.0, referenced in this PR